rules-reviewed/jboss-eap/websphere/websphere.windup.xml

```diff
@@ -0,0 +1,701 @@
+        <rule id="websphere_rules_general-7Eorol">
+            <when>
+                <javaclass references="com.ibm.websphere.sib.api.jms.JmsQueueConnectionFactory" as="default"/>
+            </when>
+            <perform>
+		<hint effort="1" severity="mandatory">
+			<message>
+				This is a websphere specific interface and needs to be migrated to standardized javax.jms.QueueConnectionFactory.
+			</message>
+			<link href="TODO" description="JMS migration"/>
+			<tag>JMS</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere_rules_general-80irik">
+            <when>
+                <javaclass references="com.ibm.websphere.sib.api.jms.JmsQueue" as="default"/>
+            </when>
+            <perform>
+                <hint effort="1" severity="mandatory">
+			<message>
+				This is a websphere specific interface and needs to be migrated to standardized javax.jms.Queue.
+			</message>
+			<link href="TODO" description="JMS migration"/>
+			<tag>JMS</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere_rules_general-82ugvt">
+            <when>
+                <javaclass references="com.ibm.websphere.sib.api.jms.JmsMsgProducer" as="default"/>
+            </when>
+            <perform>
+                <iteration>
+                    <hint effort="1" severity="mandatory">
+			<message>
+				This is a websphere specific interface and needs to be migrated to standardized javax.jms.MessageProducer.
+			</message>
+			<link href="TODO" description="JMS migration"/>
+			<tag>JMS</tag>
+                    </hint>
+                </iteration>
+            </perform>
+        </rule>
+        <rule id="websphere_rules_general-84npm">
+            <when>
+                <javaclass references="com.ibm.websphere.sib.api.jms.JmsMsgConsumer" as="default"/>
+            </when>
+            <perform>
+                <iteration>
+                    <hint effort="1" severity="mandatory">
+			<message>
+				This is a websphere specific interface and needs to be migrated to standardized javax.jms.MessageConsumer.
+			</message>
+			<link href="TODO" description="JMS migration"/>
+			<tag>JMS</tag>
+                    </hint>
+                </iteration>
+            </perform>
+        </rule>
+        <rule id="websphere_rules_general-86aij">
+            <when>
+                <javaclass references="com.ibm.websphere.sib.api.jms.JmsDestination" as="default"/>
+            </when>
+            <perform>
+                <iteration>
+                    <hint effort="1" severity="mandatory">
+			<message>
+				This is a websphere specific interface and needs to be migrated to standardized javax.jms.Destination.
+			</message>
+			<link href="TODO" description="JMS migration"/>
+			<tag>JMS</tag>
+                    </hint>
+                </iteration>
+            </perform>
+        </rule>
+        <rule id="websphere_rules_general-88qikj">
```

Comment:
These ruleset-ids should all be using dashes afaik. We agreed to use dashes. Could you update this?